### PR TITLE
Login

### PR DESCRIPTION
**Pull request by patrykmysiorski**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
 
 const App: React.FC = () => {
+  const theme = createTheme({
+    palette: {
+      primary: {
+        main: '#E1C391'
+      },
+      text: {
+        primary: '#ffffff',
+        secondary: '#ffffff'
```
Comment (qbawalat) [inline src/App.tsx:17]: pls extract values to consts

File: src/components/Form/Input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  name: string;
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  control: Control<any, Object>;
+  label: string;
+  errors: any;
+  type?: string;
+}
+
+const Input: React.FC<Props> = ({ name, control, label, errors, type, children }) => {
+  return (
```
Comment (qbawalat) [inline src/components/Form/Input/Input.tsx:17]: no need for => { return, it's an instant return

File: src/components/loginModal/LoginModal.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  const {
+    handleSubmit,
+    control,
+    formState: { errors }
+  } = useForm<Inputs>();
+  const navigate = useNavigate();
+  const [triggerLogin, { isLoading, isError }] = usePostLoginMutation();
+
+  const onSubmit: SubmitHandler<Inputs> = (data) => {
+    triggerLogin({ ...data, email: 'email2@email.pl' })
```
Comment (qbawalat) [inline src/components/loginModal/LoginModal.tsx:46]: ;o

File: src/components/loginModal/LoginModal.tsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+          <LockIcon sx={{ mr: 1, my: 0.5 }} />
+        </Input>
+        {!isLoading ? (
+          <Button className={styles.btn} type={'submit'} variant="outlined">
+            {logIn}
+          </Button>
+        ) : (
+          <Box sx={{ width: '100%', display: 'flex', justifyContent: 'center' }}>
+            <PacmanLoader color={'#E1C391'} size={15} />
+          </Box>
```
Comment (qbawalat) [inline src/components/loginModal/LoginModal.tsx:79]: pls export to component

**Review by qbawalat — APPROVED** [review]

File: src/components/Form/Input/Input.tsx (hunk 2 above)
Comment (patrykmysiorski) [reply src/components/Form/Input/Input.tsx:17]: to na przyszłość byku

File: src/components/loginModal/LoginModal.tsx (hunk 3 above)
Comment (patrykmysiorski) [reply src/components/loginModal/LoginModal.tsx:46]: TAKI BACKEND JE

File: src/components/Form/Input/Input.tsx (hunk 2 above)
Comment (qbawalat) [reply src/components/Form/Input/Input.tsx:17]: https://medium.com/swlh/stop-predicting-the-future-of-your-code-e0d403116f90

Comment (patrykmysiorski) [reply src/components/Form/Input/Input.tsx:17]: :(

File: src/components/loginModal/LoginModal.tsx (hunk 4 above)
Comment (patrykmysiorski) [reply src/components/loginModal/LoginModal.tsx:79]: right

**Pull request merged**